scm-ui/ui-components/src/repos/changesets/Changesets.stories.tsx

```diff
@@ -93,4 +97,41 @@ storiesOf("Changesets", module)
       ],
       five
     );
+  })
+  .add("With unknown signature", () => {
+    const changeset = copy(three);
+    changeset.signatures = [{
+      keyId: "0x247E908C6FD35473",
+      type: "gpg",
+      status: "NOT_FOUND"
+    }];
+    return <ChangesetRow repository={repository} changeset={changeset} />;
+  })
+  .add("With valid signature", () => {
+    const changeset = copy(three);
+    changeset.signatures = [{
+      keyId: "0x247E908C6FD35473",
+      type: "gpg",
+      status: "VERIFIED",
+      owner: "trillian",
+      contacts: [{
+        name: "Tricia Marie McMilla",
+        mail: "trillian@hitchhiker.com"
+      }]
+    }];
+    return <ChangesetRow repository={repository} changeset={changeset} />;
+  })
+  .add("With invalid signature", () => {
+    const changeset = copy(three);
+    changeset.signatures = [{
+      keyId: "0x247E908C6FD35473",
+      type: "gpg",
+      status: "INVALID",
+      owner: "trillian",
+      contacts: [{
+        name: "Tricia Marie McMilla",
+        mail: "trillian@hitchhiker.com"
+      }]
+    }];
+    return <ChangesetRow repository={repository} changeset={changeset} />;
   });
```

Comment:
I think we should one more signature story, with a change which has multiple signatures.